exonum_launcher/explorer.py

```diff
@@ -18,6 +19,15 @@ class ExecutionFailError(Exception):
     """Error raised when transaction execution fails."""
 
 
+class TxStatus(Enum):
+    """Denotes transaction status."""
+
+    Success = (enum_auto(),)
```

Comment:
Is there a reason to wrap `enum_auto()` into a tuple here? I didn't find mention of this technique in [`enum` docs](https://docs.python.org/3/library/enum.html#enum.auto).

Reply:
Hm. I suggest it was made by `black` linter.